projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -431,7 +443,16 @@ public static function get_blog_id() {
 	 * @return bool
 	 */
 	public static function has_connected_account() {
-		return get_option( self::$has_connected_account_option_name, false ) ? true : false;
+
+		// This is the primary solution.
+		$has_option = get_option( self::$has_connected_account_option_name, false ) ? true : false;
```

Comment:
```suggestion
		$has_option = get_option( self::$has_connected_account_option_name, false );
```

Also... :)

Reply:
I prefer ensuring that it's true or false instead of 0, 1, null or whatever could be put in there...  That's probably just my java background though.  I can change if you feel strongly about it, but my vote is to keep it.

Reply:
My JS background would write it has:
```
$has_option = !!get_option( self::$has_connected_account_option_name, false );
```
😬

Joke aside, `get_option` will return `false` if not set, and a boolean if it was set... So not sure of your intent there...

Reply:
`get_option()` might return a boolean if it's set, but it might not depending on what the data is...

```
 * @return mixed Value of the option. A value of any type may be returned, including
 *               scalar (string, boolean, float, integer), null, array, object.
 *               Scalar and null values will be returned as strings as long as they originate
 *               from a database stored option value. If there is no option in the database,
 *               boolean `false` is returned.
```

Reply:
Yes but you set it as a boolean!

Reply:
Doesn't stop the site admin from changing it to something else...

Reply:
I see... But if the admin changes to something else we can't even use it as it is no more a source of truth 😅

Reply:
But I see your point... Let's keep it then

Reply:
Thanks!